src/components/Comments/Comments.jsx

```diff
@@ -21,11 +24,40 @@ const Comments = ({article_id}) => {
         })
     },[])
 
+    const handleChange = (event) => {
+        setNewComment(event.target.value);
+    }
+
+    const postComment = () => {
+        setIsPostingComment(true);
+        setPosted(false);
+        if (!newComment){
+            console.log('no')
+            return;
+        }
+        requests.addComment(article_id, {"username":user, "body":newComment})
+        .then((data) => {
+            setComments([data,...comments]);
+            setIsPostingComment(false);
+            setPosted(true);
+        })
+        .catch((err) => {
+            if(err.code === "ERR_NETWORK"){
+                alert('Internet connection is offline..')
+            }
+        })
+    }
+
     if (loadingComments){
         return <p>Loading comments...</p>
     }else{
         return <div>
-                <ul>
+            <label htmlFor="commentInput">Comment:</label>
+            <input type="text" id="commentInput" onChange={handleChange}></input>
```

Comment:
input tags are designed for single lines of text. Our commenters may have a lot to say and want to add line breaks.